spharpy/spherical.py

```diff
@@ -248,6 +248,104 @@ def n3d_to_sn3d_norm(n):
     return 1 / np.sqrt(2 * n + 1)
 
 
+def renormalize(data, channel_convention, current_norm, target_norm, axis=-2):
+    """
+    Renormalize either an array of SH coefficients or SH basis functions.
+
+    Parameters
+    ----------
+    data : ndarray
+        Data which should be renormalized, either SH
+        coefficients or SH bases
+    channel_convention : str
+        Channel convention of the data which should be renormalized
+    current : str
+        Current channel convention
+    target : str
+        Desired channel convention
+    axis : integer
+        Axis along which the channel convention should be changed
+
+    Returns
+    -------
+    data : float or complex, ndarray
+        Renormalized data
+    """
+    if target_norm not in ["n3d", "maxN", "sn3d"]:
+        raise ValueError("Invalid normalization, has to be 'sn3d', "
+                         f"'n3d', or 'maxN', but is {target_norm}")
+    acn = np.arange(data.shape[axis])
+
+    if channel_convention == "fuma":
+        orders, _ = fuma_to_nm(acn)
+    else:
+        orders, _ = acn_to_nm(acn)
+
+    # prepare helper for reshaping
+    shape = [1] * data.ndim
+    shape[axis] = -1
+
+    if current_norm == 'n3d':
+        if target_norm == "sn3d":
+            data *= n3d_to_sn3d_norm(orders).reshape(shape)
+        elif target_norm == "maxN":
+            data *= n3d_to_maxn(acn).reshape(shape)
+
+    if current_norm == 'sn3d':
+        # convert to n3d
+        data /= n3d_to_sn3d_norm(orders).reshape(shape)
+        if target_norm == "maxN":
+            data *= n3d_to_maxn(acn).reshape(shape)
+
+    if current_norm == 'maxN':
+        # convert to n3d
+        data /= n3d_to_maxn(acn).reshape(shape)
+        if target_norm == "sn3d":
+            data *= n3d_to_sn3d_norm(orders).reshape(shape)
+    return data
+
+
+def change_channel_convention(data, current, target, axis=-2):
```

Comment:
```suggestion
def change_channel_convention(data, current, target, axis):
```
Same comment as above